tests/ceph_volume/test_ceph_volume_lvm_cli_validations.py

```diff
@@ -0,0 +1,42 @@
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Passing incorrect arguements to ceph-volume lvm
+    prepare, activate and create command should not
+    cause crashes. User understandable error/warning shuld be
+    displayed
+    """
+    # Get configs
+    configs = kw.get("config")
+
+    # Get specs from config
+    data = configs.get("device_path")
+    osd_id = configs.get("osd_id")
+    osd_fsid = configs.get("osd_fsid")
+
+    # Get the installer node
+    node = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Run the command lvm prepare
+    out = CephAdm(node).ceph_volume.lvm.prepare(data)[1]
```

Comment:
Create single `CephAdm` object and call all time

Reply:
Done